app/controllers/idv/enter_password_controller.rb

```diff
@@ -143,6 +144,7 @@ def init_profile
       end
 
       if profile.active?
+        attempts_api_tracker.idv_enrollment_complete(reproof:)
```

Comment:
Could this be moved below `UserAlerts::AlertUserAboutAccountVerified.call`? I think it might be preferable to run the other things first (in terms of importance).

Reply:
sure, added in https://github.com/18F/identity-idp/pull/12132/commits/a39cefbb15d37a56ecf3b2b34279c73b091f7be6